### PR TITLE
[IMP][workflow_task] Remove the technical field ref_object from the a…

### PR DESCRIPTION
**Pull request by lmignon**
…dvanced search

ping @sbidoul @adrienpeiffer

### CODE REVIEW

File: workflow_task/models/task.py [hunk 1]
```diff
@@ -73,6 +73,17 @@ def _select_objects(self):
...
 
+    
+    def fields_get(self, cr, user, allfields=None, context=None,
+                   write_access=True, attributes=None):
+        res = super(Task, self).fields_get(
+            cr, user, allfields, context, write_access, attributes)
+        # remove ref_object from searchable field into the advanced search
+        # since the field to use is ref_object_name
+        if 'ref_object' in res:
+            res['ref_object']['searchable'] = False
```
Comment (adrienpeiffer) [inline workflow_task/models/task.py:83]: You can not define `searchable=False` directly on the field ?

Comment (lmignon) [inline workflow_task/models/task.py:83]: @adrienpeiffer unfortunately no :sob:  `searchable`is defined as a python property on `fields.Field` https://github.com/odoo/odoo/blob/9.0/openerp/fields.py#L675
You can see at line 125 how odoo compute the list of attributes returned by the function field_get https://github.com/odoo/odoo/blob/9.0/openerp/fields.py#L125
The only solution I've found is override `fields_get` to change the value returned to the JS by the server

**Comment by adrienpeiffer** [comment]
Ok :smile: can you check travis build ?

**Comment by lmignon** [comment]
@adrienpeiffer LINT_CHECK fixed, 9.0-workflow-task-sbi rebased on origin/9.0 and 9.0-workflow-taks-filter-search-field-lmi rebased on 9.0-workflow-task-sbi...
:white_check_mark:

**Comment by adrienpeiffer** [comment]
Thanks @lmignon

**Pull request merged**